test/plugin/scenarios/logger-log4j2-scenario/support-version.list

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+2.14.0
+2.13.0
+2.12.0
+2.11.0
+2.10.0
+2.9.0
+#2.8
```

Comment:
I wasn't able to get these versions to test correctly. I think it has to do with the dependencies Spring Boot pulls in though and not that the plugin doesn't work for them.